browser/brave_wallet/brave_wallet_service_delegate_impl.cc

```diff
@@ -519,7 +519,7 @@ std::string BraveWalletServiceDelegateImpl::GetActiveOriginInternal() {
   content::WebContents* contents = GetActiveWebContents();
   return contents ? contents->GetMainFrame()
                         ->GetLastCommittedURL()
-                        .GetOrigin()
+                        .DeprecatedGetOriginAsURL()
```

Comment:
`.GetLastCommittedOrigin().Serialize()`

Reply:
There's an important doc we should follow now: https://chromium.googlesource.com/chromium/src/+/main/docs/security/origin-vs-url.md

Reply:
Fixed in https://github.com/brave/brave-core/commit/02855182fa0d19d311ec82c3ac847310c2899106 as well

Reply:
Soo... this is interesting: apparently, while `contents->GetMainFrame()->GetLastCommittedURL().DeprecatedGetOriginAsURL().spec()` returns a valid string (e.g. `https://d.com:38831/`), doing `contents->GetMainFrame()->GetLastCommittedOrigin().Serialize()` returns `nullptr`!

Even `contents->GetMainFrame()->GetLastCommittedOrigin().GetURL().spec()` (which I mistakenly added in the branch instead of your suggestion) returns an empty string, so clearly there's something different here when using `GetLastCommittedURL()` compared to `GetLastCommittedOrigin()`

For now, I restored the original code in https://github.com/brave/brave-core/commit/2501151fa9700f15f438cfc683dd9ba2e380a649

Reply:
Fixed in 3c06afb7ae381abfdf9e9047607fc6a8c4b4b0c0